ring-core/test/ring/middleware/session/test/cookie.clj

```diff
@@ -56,3 +56,16 @@
   (is (thrown-with-msg?
        AssertionError #"the secret key must be exactly 16 bytes"
        (cookie-store {:key (.getBytes "012345678901234567890")}))))
+
+; setup for serializing/deserializing Instant.
+(defmethod print-method java.time.Instant
+  [dt out]
+  (.write out (str "#foo/instant \"" (.toString dt) "\"")))
```

Comment:
At the end of each `defmethod`, `defn` or `deftest` there should be a blank line.